enigma-tools-m/Cargo.toml

```diff
@@ -23,13 +23,13 @@ serde_std = { package = "serde", version = "1.0", default-features = false, opti
 serde_json_std = { package = "serde_json", version = "1.0", optional = true }
 
 
-ethabi_sgx = { package = "ethabi", git = "https://github.com/enigmampc/ethabi.git", rev = "sgx-v6.1.0-v1.0.7", optional = true }
+ethabi_sgx = { package = "ethabi", git = "https://github.com/enigmampc/ethabi.git", rev = "6.1.0-sgx-1.0.8", optional = true }
+# we don't use sgx on ethereum-types
 ethereum_types_sgx = { package = "ethereum-types", git = "https://github.com/enigmampc/primitives.git", rev = "sgx-v0.4.0", default-features = false, optional = true }
-rmp_serde_sgx = { package = "rmp-serde", git = "https://github.com/enigmampc/msgpack-rust.git", rev =  "sgx-v1.0.7", optional = true }
-sgx_tstd = { git = "https://github.com/baidu/rust-sgx-sdk.git", rev = "v1.0.7", optional = true }
-serde_sgx = { package = "serde", git = "https://github.com/baidu/rust-sgx-sdk.git", rev = "v1.0.7", default-features = false, optional = true }
-serde_json_sgx = { package = "serde_json", git = "https://github.com/baidu/rust-sgx-sdk.git", rev = "v1.0.7", optional = true }
-
+rmp_serde_sgx = { package = "rmp-serde", git = "https://github.com/enigmampc/msgpack-rust.git", rev =  "0.6.0-sgx-1.0.8", optional = true }
```

Comment:
The fact that the latest tag is `0.6.0` doesn't mean that this is what we use.
if you look at that tag you'll see it's from 2015.

and this one specifically is "problematic" in that sense because it has multiple crates with different versions: https://github.com/enigmampc/msgpack-rust/blob/0.6.0-sgx-1.0.8/rmp-serde/Cargo.toml
https://github.com/enigmampc/msgpack-rust/blob/0.6.0-sgx-1.0.8/rmp/Cargo.toml